modules/common.nix

```diff
@@ -164,7 +164,36 @@
       lib.mapAttrsRecursiveCond (x: ! (lib.isDerivation x))
       (_: fakeDerivation: fakeDerivation.meta.element or "this catalog doesn't add element to meta")
       newFakeCatalog;
+    # turn packagesList into a backwards compatible manifest.json
+    manifestJSON = builtins.toJSON {
+      version = 2;
+      elements =
+      lib.flatten
+        (lib.mapAttrsToList (
+          _: systemPackages:
+            lib.mapAttrsToList (system: stabilityPackages:
+              lib.mapAttrsToList (
+                stability: packagePackages:
```

Comment:
How does this work with nested packages? `python3Packages.requests`?

Reply:
Hmm good point, I haven't handled that as a whole. Using the current format I'm expecting, how would we differentiate between:
`nixpkgs-flox.python3Packages.requests = { version = "1.12" };`
and
`nixpkgs-flox.VSCode.settings = { key = "value"; }`